docs.sh

```diff
@@ -0,0 +1,23 @@
+set -e
+
+rimraf esdoc
+
+exec 5>&1
+OUT=$(esdoc -c docs/esdoc-config.js|tee /dev/fd/5)
+
+cp docs/favicon.ico esdoc/favicon.ico
+cp docs/ROUTER.txt esdoc/ROUTER
+
+node docs/run-docs-transforms.js
+node docs/redirects/create-redirects.js
+
+rimraf esdoc/file esdoc/source.html
+
+set +e
+GREP_RESULT=$(echo "$OUT" | grep -c 'could not parse the following code')
+set -e
+
+if [ "$GREP_RESULT" -ge 1 ]; then
```

Comment:
Did you test this on GitHub Actions already? Just to check if it fails the same way as locally

Reply:
I haven't but I'm running the same os and these are base tools, and it succeeded in the ci, it should fail properly if there is a syntax error